packages/daimo-common/src/daimoLink.ts

```diff
@@ -163,61 +182,64 @@ function parseDaimoLinkInner(link: string): DaimoLink | null {
       suffix = link.substring(prefix.length);
     }
   }
-  if (suffix == null) return null;
+  if (suffix == null) return defaultError;
 
   const parts = suffix.split("/");
 
   switch (parts[0]) {
     case "account": {
-      if (parts.length !== 2) return null;
+      if (parts.length !== 2) return defaultError;
       const account = parts[1];
       return { type: "account", account };
     }
     case "request": {
-      if (parts.length !== 4) return null;
+      if (parts.length !== 4) return defaultError;
       const recipient = parts[1];
       const dollarNum = parseFloat(zDollarStr.parse(parts[2]));
-      if (!(dollarNum > 0)) return null;
+      if (!(dollarNum > 0)) return defaultError;
       const dollars = dollarNum.toFixed(2) as DollarStr;
       const requestId = `${BigInt(parts[3])}` as BigIntStr;
 
-      if (dollars === "0.00") return null;
+      if (dollars === "0.00") return defaultError;
```

Comment:
tbh i think this whole file (`daimoLink.ts`) was fine before-- we just need handling in the event `parseDaimoLink` returns null

we can move the `exp+daimo` development links check: that's the only type of link i can think of we'd want to *silently* ignore

in all other cases, if a link is passed to the app as a deeplink:
- call parseDaimoLink
- if we get a parsed result, display
- otherwise, display the error screen